_field-types/mapping-parameters/enabled.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: enabled
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 40
+has_children: false
+has_toc: false
+---
+
+# Enabled
+
+The `enabled` parameter allows you to control whether OpenSearch parses the contents of a field. This parameter can be applied to the top-level mapping definition and to object fields.
+
+The `enabled` parameter accepts the following values. 
+
+Parameter | Description
+:--- | :---
+`true` | The field is parsed and indexed. Default is `true`.
+`false` | The field is not parsed or indexed, but still retrievable from the `_source` field. When `enabled` is set to `false`, OpenSearch stores the field's value in the `_source` field but does not index or parse its contents. This can be useful for fields that you want to store but do not need to search, sort, or aggregate on.
```

Comment:
```suggestion
`false` | The field is not parsed or indexed but is still retrievable from the `_source` field. When `enabled` is set to `false`, OpenSearch stores the field's value in the `_source` field but does not index or parse its contents. This can be useful for fields that you want to store but do not need to search, sort, or aggregate on.
```